tests/testthat/helper-unit-mocks.R

```diff
@@ -1,67 +1,75 @@
-mock_coverage_set <- function(id = 1:4,
+db <- tryCatch({
+  # will work during devtools::check
+  fakerbase::fb_load("montagu", package = "vimpact")
+}, error = function(cond) {
+  # will work during devtools::test
+  return(fakerbase::fb_load("montagu", path = "../../inst"))
```

Comment:
Could do
```
db <- fakerbase::fb_load("montagu", path = system.file(package = "vimpact", mustWork = TRUE))
```
here instead. Agree it would be preferable though if there is a way to get that working with `package = "vimapact"`. @richfitz might have some clever idea when he gets back. But from some googling around looks like it might not be possible for `system.file` to work in this context https://community.rstudio.com/t/how-is-system-file-supposed-to-work-when-a-package-is-loaded-using-load-all-or-devtools-test/101979

Reply:
Sorry perhaps my comment wasn't very clear; under the hood `fakerbase::fb_load("montagu", package = "vimpact")` just calls through to `system.file`, which works during `devtools::check` but not during `devtools::test`, because during the former `system.file(package = "vimpact")` is poly-filled to look in the temp directory where the package installation has been simulated, but during the latter it is looking in your R library for the real installed package.

Reply:
But my grasp of all the packaging/testing protocols is not 100% and yes, maybe Rich has some clever way around this